package.json

```diff
@@ -34,7 +34,13 @@
   },
   "homepage": "https://github.com/collegevine/purescript-elmish-hooks#readme",
   "devDependencies": {
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
+    "enzyme": "^3.11.0",
+    "global-jsdom": "^8.4.0",
+    "jsdom": "^16.5.1",
```

Comment:
Note to self: It was difficult to figure this out: I had to install `jdom@16` with the latest `global-jsdom`.